backend/src/controller/controller.js

```diff
@@ -7,9 +7,14 @@ const jwt = require('jsonwebtoken');
 exports.register = async (req, res) => {
     const { name, email, password } = req.body
 
+    console.log(req.body)
```

Comment:
Hey Quintus, make sure to remove your console.logs before committing to the master branch!